src/scss/partials/filterList/_FilterList.scss

```diff
@@ -0,0 +1,2 @@
+.filterList {
```

Comment:
jeżeli nie ma potrzeby stylowania tak ja tutaj to nie potrzebujemy tworzyć nowego pliku. To nic nie wnosi :)